### PR TITLE
Feature/module dashboard

### PR DESCRIPTION
**Pull request by srslie**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [x] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- adds display of module inputs on module dashboard
- adds button/form/patch to update module inputs

## *What are the relevant tickets?*
Closes _#5

## *Bugs:*
- patch gives successful response but does not trigger re-pull of updated data, and a hard refresh also doesn't
- on added modules with no input values, blank pages show up, should put in validation
- needs testing

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have run Lighthouse/Wave checks for accessibility
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: context/apiCalls.js [hunk 1]
```diff
@@ -38,3 +38,23 @@ export const deleteData = (type, id) => {
...
+  return fetch(url, settings)
+  .then((response) => {
+      console.log('PATCHRESPONSE', response.status)
+    if (!response.ok) {
+      throw Error(response.statusText);
+    }
+    return response.json();
+  })
+  .catch((error) => console.error(error));
+}
```
Comment (lbmerchant93) [inline context/apiCalls.js:60]: Great job throwing the patch in!

File: pages/editModuleForm.js [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        <div className={styles.timeMeter}>
+          <div className={styles.range} style={{width: `${range}`}} ></div>
+          <div className={styles.timeUsed} style={{width: `${totalInputPercent}`, backgroundColor: `${barColor}`}} ></div>
+        </div>
+
+      </div>
+
+      </div>
+  )
+}
```
Comment (lbmerchant93) [inline pages/editModuleForm.js:258]: This whole thing is impressive!

**Review by lbmerchant93 — APPROVED** [review]
Looks dope! Great job figuring this all out!

**Review by alia-peterson — APPROVED** [review]

**Review by camaragon — APPROVED** [review]
You really outdid yourself here Alice!

**Pull request merged**